### DISCUSSION TITLE
v2: git repo and PAT

### DISCUSSION THREAD

**Discussion by Skorfulose** [almenscorner/intunecd-monitor#33] [Q&A]
Hi Tobias and community,

the docs for v2 currently just give an example for using a GitHub repo. Am I right to assume Azure DevOps repos work equally well?

And another question:
Using PATs, repo access is always bound to my personal account. What if I'm hit by a bus? If in a larger organization, such things should be non-personal. Like it was with the ADO pipeline approach. I am not keen on the pipeline per se, but do you have any thoughts on alternative authentication to GitHub / GitLab / Azure DevOps, etc?

Kind regards, and keep up the good work!
Thomas

**Comment by almenscorner** [comment]
I do not know of any other "universal" way to connect to a git repo really.

If you are hit by a bus (let's hope not), another admin can just override the PAT configured in IntuneCD Monitor by editing the tenant and changing the PAT. It is also retrievable from the Key Vault.
